decidim-core/config/locales/ca.yml

```diff
@@ -202,6 +202,7 @@ ca:
     newsletter_mailer:
       newsletter:
         note: Has rebut aquest missatge perquè estàs subscrit a butlletins de notícies a %{organization_name}. Pots canviar la configuració a la teva <a href="%{link}">pàgina de notificacions</a>.
+        see_on_website: No veus correctament aquest missatge? Mira'l al <a href="%{link}">lloc web</a>.
```

Comment:
Please, add locales for only English. Translations are handled via Crowdin, and if we add some translations manually directly to the code the integration gets weird.